tests/archiver.rs

```diff
@@ -587,8 +605,8 @@ const OVERRIDES: [Override; 19] = [
         "china-national-standard-gb-t-7714-2015-author-date",
         "gb-7714-2015-author-date",
     ),
-    Override::first("chicago-fullnote-bibliography", "chicago-fullnotes"),
-    Override::first("chicago-note-bibliography", "chicago-notes"),
+    Override::first("chicago-notes-bibliography", "chicago-notes"),
+    Override::first("chicago-shortened-notes-bibliography", "chicago-shortened-notes"),
```

Comment:
Wonder if we can add fullnotes back as an alias to soften the impact of this change?